### PR TITLE
Feature/SC - 7083 get school id from admin

### PR DESCRIPTION
**Pull request by JenyMzo**
# Description

Creates input to type the school id in the school administration for users who have admin permissions.

## Links to Tickets or other pull requests
https://github.com/hpi-schul-cloud/schulcloud-server/pull/1833
https://ticketsystem.hpi-schul-cloud.org/browse/SC-7083

## Changes
Adds school id input text input to type the school id in the school administration view.

## Screenshots of UI changes
Screenshot 2020-10-15 at 15 47 57

## Approval for review
- [X] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: controllers/topics.js [hunk 1]
```diff
@@ -475,7 +476,7 @@ router.get('/add/neweditor', async (req, res, next) => {
 		title: '',
 	});
 
-	res.redirect(`/courses/${req.params.courseId}/topics/${lesson._id}?edtr=true`);
+	res.redirect(`/courses/${req.params.courseId}${lesson._id}?edtr=true`);
```
Comment (virgilchiriac) [inline controllers/topics.js:479]: why this change?

**Review by virgilchiriac — APPROVED** [review]

**Review by Ojself — APPROVED** [review]
